src/software/jetson_nano/services/network/network.h

```diff
@@ -40,7 +40,8 @@ class NetworkService
 
    private:
     // Constants
-    static constexpr float PROTO_LOSS_WARNING_THRESHOLD = 0.1f;
+    static constexpr float PROTO_LOSS_WARNING_THRESHOLD          = 0.1f;
+    static constexpr unsigned int ROBOT_STATUS_BROADCAST_RATE_HZ = 60;
```

Comment:
If this is how many robot status messages we send per second, then I think we can reduce it further. I think 5-10 is likely good enough...

Reply:
Think that would be too low. sensor fusion uses breakbeam_tripped in power status to determine if we have the ball. we previously used 60 maybe 20-30 is sufficient

Reply:
ya with all of the improvements we've made to reduce congestion, it might be fine around 30. If we find that we still experience problems, we can reduce it further, but send a message everytime breakbeam status changes